storageClient/api.go

```diff
@@ -80,9 +85,36 @@ type PrimeResponseBody struct {
 
 type ChallengeRequest struct {
 	EmailEACPChallenge EmailEACPChallengeRequest `json:"email_eacp"`
+	TozIDEACPChallenge TozIDEACPChallengeRequest `json:"tozid_eacp"`
 }
 
 type EmailEACPChallengeRequest struct {
 	TemplateName string `json:"template_name"`
 	OTPRequest
 }
+
+// TozIDEACPChallengeRequest wraps parameters needed to activate a TozID EACP on a note.
+type TozIDEACPChallengeRequest struct {
+	NoteID        string
+	IDString      string
+	ExpirySeconds int64 `json:"expiry_seconds"`
+}
+
+// ChallengeResponse wraps parameters for all activated challenges on a note.
+type ChallengeResponse struct {
+	EmailEACPChallenge *EmailEACPChallengeResponse `json:"email_eacp,omitempty"`
+	TozIDEACPChallenge *TozIDEACPChallengeResponse `json:"tozid_eacp,omitemtpy"`
+}
+
+type EmailEACPChallengeResponse struct {
+	EmailAddress string `json:"email_address"`
+	TemplateName string `json:"template"`
+	ProviderLink string `json:"provider_link"`
+	OTP          string `json:"otp"`
```

Comment:
We definitely don't want to return the OTP or provider link from this EACP -- it should only be delivered via the email channel out of band of this request flow.

Reply:
As mentioned in the API spec, I'm actually thinking we don't want to respond with any of this information. Happy to discuss, but in general this should be kept to a very minimum.